lighthouse-core/test/audits/audit-test.js

```diff
@@ -20,6 +20,30 @@ class B extends Audit {
   static audit() {}
 }
 
+class PassOrFailAudit extends Audit {
+  static get meta() {
+    return {
+      id: 'pass-or-fail',
+      title: 'Passing',
+      failureTitle: 'Failing',
+      description: 'A pass or fail audit',
+      requiredArtifacts: [],
+    };
+  }
+}
+
+class NumericAudit extends Audit {
+  static get meta() {
+    return {
+      id: 'numeric-time',
+      title: 'Numbersssss',
+      description: '01000000001011011111100001010100',
```

Comment:
`@-�T`?

c'mon gotta have some next level messaging in here :)

Reply:
`01101101011001010110111101110111`

Reply:
not everything's a character :P

`new Float32Array(new Uint32Array([0b01000000001011011111100001010100]).buffer)[0]`

Reply:
touche :)